### PR TITLE
remove dependency on multimap

### PR DESCRIPTION
**Pull request by elharo**
@suztomo I noticed it was pretty easy to remove this multimap from our public API.

### CODE REVIEW

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java [hunk 1]
```diff
@@ -93,21 +105,14 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
...
   }
-
+  
   /**
-   * Returns mapping from the Maven coordinates to {@link ClassPathEntry}. The keys are the
-   * coordinates of the direct dependencies of the root nodes in {@link #dependencyPaths}. The
-   * values are all {@link ClassPathEntry}s in the subtree of the key.
+   * Returns the classpath entries for the specified coordinates.
+   * 
+   * TODO: Explain why this is plural. Shouldn't there be exactly one?
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java:112]: As in the previous javadoc, the values are all {@link ClassPathEntry}s in the subtree of the key.

Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java:114]: Probably not though we can revisit. The focus is on not passing a bunch of weird data structures around. Instead I'd like public classes that represent our entities (ClassFile, ClassPathEntry, ClassPath, Artifact, Dependency, etc.) and operations on those.

Comment (suztomo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java:114]: That's good. Would you revert the javadoc update? The previous javadoc explains why the values are plural.

Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java:99]: I'm not sure it did. Or rather what it said didn't match the code. Notice that this returns class path entries, **not** dependency paths.

Comment (suztomo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java:99]: How about "Returns the class path entries in the subtree of the specified coordinates"?

Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java]: done

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java [hunk 2]
```diff
@@ -93,21 +92,24 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
...
     return message.toString();
   }
-
+  
   /**
-   * Returns mapping from the Maven coordinates to {@link ClassPathEntry}. The keys are the
-   * coordinates of the direct dependencies of the root nodes in {@link #dependencyPaths}. The
-   * values are all {@link ClassPathEntry}s in the subtree of the key.
+   * Returns the classpath entries whose transitive dependencies contain
+   * the artifact with the specified coordinates.
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java:98]: It's the other way around. The artifact with the specified coordinates contains the class path entries as (transitive and direct) dependencies.

Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java]: fixed

**Pull request merged**